phoenix-flume-base/phoenix5-flume/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>phoenix-flume-base</artifactId>
+        <groupId>org.apache.phoenix</groupId>
+        <version>6.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>phoenix5-flume</artifactId>
+    <name>Phoenix5 - Flume</name>
+
+    <properties>
+        <top.dir>${project.basedir}/..</top.dir>
+        <phoenix.version>5.1.0-SNAPSHOT</phoenix.version>
+        <hbase.version>2.2.4</hbase.version>
```

Comment:
This version handling is messy.

I think that the best option would be  defining hadoop-two, hadoop-three, hbase-one, hbase-two in the top level pom, and using those in the modules (i.e. <hadoop.version>${hadoop-three.version}</hadoop.version>) That way both versions can be overriden at build time.

In fact the same could be done for phoenix5.version and phoenix4.version.